App/pom.xml

```diff
@@ -157,6 +171,16 @@
             <artifactId>javafx-fxml</artifactId>
             <version>${javafxVersion}</version>
         </dependency>
+        <!--<dependency>
```

Comment:
this is commented, so no need to add this?

Reply:
The commented out block was present and was removed recently. There are multiple sections in files including pom.xml, module-info.java and GluonChat.java which has commented out lines for running ScenicView. I like to keep them as it helps debugging easier. However, I can remove them if its a problem.

Reply:
Ok, that's a good reason to keep them (in comments).